LibraryApp/src/app/components/LogoutButton/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import CustomButton from '@components/CustomButton';
+import authActions from '@redux/auth/actions';
+
+import styles from './styles';
+
+const LOGOUT_TEXT = 'LOGOUT';
+
+class LogoutButton extends Component {
+  handleLogout = () => {
```

Comment:
I rather keep it single line in this cases `handleLogout = () => this.props.dispatch(authActions.logout());`